.github/workflows/builder.yaml

```
@@ -69,5 +69,5 @@ jobs:
               with:
                   name: ${{ inputs.example-app }}-${{inputs.platform}}-${{ inputs.build-type }}
                   path: out/*/*/artifact
-                  if-no-files-found: error
+                  if-no-files-found: ignore
```

I think warning might be the better option to at least draw eyes to this to make sure it is deliberate 
```suggestion
                  if-no-files-found: warning
```